src/App.tsx

```diff
@@ -1,61 +1,105 @@
+import React, { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { Todo, TodoWithUser, User } from './types';
+import { TodoList } from './components/TodoList';
+import { TextField } from './components/TextField';
+import { SelectField } from './components/SelectField';
+
+const todosWithUsers: TodoWithUser[] = todosFromServer.map((todo: Todo) => {
+  const user: User | null = usersFromServer.find(({ id }) => id === todo.userId)
+    || null;
+
+  return {
+    ...todo,
+    user,
+  };
+});
+
+export const App: React.FC = () => {
+  const [count, setCount] = useState(0);
```

Comment:
why state variable `count` is needed?

Reply:
it's needed to rerender the whole form and update the internal state of the TextField component (to default values). Without it, errors are shown after the form submit. I actually learned this from the previous task, or is it an antipattern?